src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
 /**
+ *
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+
+  if (days < 3) {
+    return cost * days;
+  }
+
+  if (days < 7) {
+    return cost * days - 20;
+  }
+
+  if (days >= 7) {
```

Comment:
Similarly, the condition `if (days >= 7)` can be changed to `else` since it covers all remaining cases after the previous conditions.